docs/tutorials/control/flutter-app.md

```diff
@@ -0,0 +1,736 @@
+---
+title: "Build a Flutter App that Integrates with Viam"
+linkTitle: "Build a Flutter App"
+type: "docs"
+description: "Use Viam's Flutter SDK to build a custom mobile app to show your machines and their components."
+webmSrc: "/tutorials/flutter-app/demo.webm"
+mp4Src: "/tutorials/flutter-app/demo.mp4"
+videoAlt: "An example Viam-integrated Flutter app."
+tags: ["sdk", "flutter"]
+authors: ["Clint Purser"]
+languages: ["flutter"]
+viamresources: ["base"]
+level: "Intermediate"
+date: "2024-01-17"
+cost: "0"
+---
+
+<div class="td-max-width-on-larger-screens">
+ <div class="alignleft" style="max-width:250px; margin-right:40px">
+  {{<gif webm_src="/tutorials/flutter-app/demo.webm" mp4_src="/tutorials/flutter-app/demo.mp4" alt="Rendering of the mobile app. Log in is clicked, then all Clint's locations are shown, Clint's Desk it clicked, all its smart machines are listed, and then desk-robot is clicked, showing a list of components and services belonging to that smart machine.">}}
+  </div>
+</div>
+
+Flutter is Google's user interface toolkit for building applications for mobile, web, and desktop from a single codebase.
+If the existing [Viam mobile app](/fleet/#the-viam-mobile-app) doesn't fit your needs, you can use Viam's [Flutter SDK](https://flutter.viam.dev/) to build a custom app to interact with your smart machines that run on Viam.
+
+This tutorial guides you through creating a mobile app that shows your machines and their components.
+As you work through this project you will learn the following:
+
+- The basics of how Flutter works
+- How to create _layouts_ in Flutter
+- How to connect user interactions (such as button presses) to app behavior
+- The basics of using Viam's Flutter SDK
+
+## Requirements
+
+You do not need any hardware for this tutorial other than a computer running macOS or a 64-bit Linux operating system.
```

Comment:
why not windows?

Reply:
My impression is that it's because this tutorial covers development for iOS, which cannot be done on Windows unless you run a virtual machine? See also question below

Reply:
https://www.reddit.com/r/FlutterDev/comments/1838hcb/can_you_build_ios_apps_on_windows_with_flutter/ seems to confirm that you'd need additional tooling to build an ios app on windows.